tests/scripts/analyze_outcomes.py

```diff
@@ -238,6 +238,13 @@ def do_analyze_driver_vs_reference(results: Results, outcome_file, args):
                     # doesn't, we have a PASS vs SKIP mismatch.
                     'Check mbedtls_calloc overallocation',
                 ],
+                'test_suite_ctr_drbg': [
+                    # Threaded tests will be skipped on non SHA512 builds,
+                    # but these do not necessarily match between reference and
+                    # driver components.
+                    'CTR_DRBG Threads: no reseed',
```

Comment:
The issue appears to be that driver builds are not necessarily supporting SHA512 on the same profiles as the reference build, thus there was a skip / pass mismatch here. I can file an issue to follow this up, but I must admit I'm not entirely sure where to start on this one.

Reply:
The default entropy size collected during seeding depends on the hashes available.
> //#define MBEDTLS_CTR_DRBG_ENTROPY_LEN               48 /**< Amount of entropy used per seed by default (48 with SHA-512, 32 with SHA-256) */

This makes sense, because the Mbed TLS entropy collector uses hashes. In the tests we use a mock entropy function, but the entropy length change will still affect the output. If we set the entropy length manually in the tests (`mbedtls_ctr_drbg_set_entropy_len()`), the dependance on hashes will be removed.